tools/git-version.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.ajoberstar:grgit:1.9.2'
+    }
+}
+
+import org.ajoberstar.grgit.Grgit
+
+ext {
+    git = Grgit.open(currentDir: projectDir)
+    gitVersionName = ""
+    gitVersionName = git.describe()
```

Comment:
describe only returns the weird hash when executing it in a commit that has no tag. Doing it in a tag returns just the tag name, which is fine.